Update yaml_utils.cpp

fmeisel opened the pull request:
# fix fme 2021-04-29 - sd=000000  leads to an error because check is just < or >, not <= or >

aligungr commented:
@fmeisel

Thanks for the pull request.

I'm kindly ask you to rollback the changes you made in the config files, which are:
- custom-gnb.yaml
- custom-ue.yaml
- free5gc-ue.yaml
- open5gs-ue.yaml

Because making changes in the config files may have breaking side effects for the people who use them.

And then I will merge it.
Thanks again for the PR

Regards

fmeisel commented:
@aligungr 
i changed back the config default. 

can you reject the change in "yaml_utils.cpp" that is not needed.
=> Sorry i am not a developer, so i don´t know how to delete this change

Those are important.  (which i changed)
src/gnb.cpp 
src/ue.cpp

Regards

aligungr commented:
I merged it to the master, thanks
